src/test/java/net/snowflake/ingest/SimpleIngestIT.java

```diff
@@ -315,4 +312,60 @@ private void verifyDefaultUserAgent(
       }
     }
   }
+
+  @Test
+  public void testConfigureClientHappyCase() throws Exception {
+    final String userAgentSuffix = "kafka-provider/NONE";
+    SimpleIngestManager manager = TestUtils.getManager(pipeName, userAgentSuffix);
+    ConfigureClientResponse configureClientResponse = manager.configureClient(null);
+    assertEquals(0L, configureClientResponse.getClientSequencer().longValue());
+  }
+
+  @Test
+  public void testConfigureClientNoPipeFound() throws Exception {
+    final String userAgentSuffix = "kafka-provider/NONE";
+    SimpleIngestManager manager = TestUtils.getManager("nopipe", userAgentSuffix);
+    try {
+      manager.configureClient(null);
+    } catch (IngestResponseException exception) {
+      assertEquals(
+          exception.getMessage(),
+          "HTTP Status: 404 ErrorBody: {\n"
+              + "  \"data\" : null,\n"
+              + "  \"code\" : \"390404\",\n"
+              + "  \"message\" : \"Specified object does not exist or not authorized. Pipe not found\",\n"
+              + "  \"success\" : false,\n"
+              + "  \"headers\" : null\n"
+              + "}");
+    }
+  }
+
+  @Test
+  public void testGetClientStatusHappyCase() throws Exception {
+    final String userAgentSuffix = "kafka-provider/NONE";
+    SimpleIngestManager manager = TestUtils.getManager(pipeName, userAgentSuffix);
+    manager.configureClient(null);
+    ClientStatusResponse clientStatusResponse = manager.getClientStatus(null);
+    assertEquals(0L, clientStatusResponse.getClientSequencer().longValue());
+    assertEquals(null, clientStatusResponse.getOffsetToken());
+  }
+
+  @Test
+  public void testGetClientStatusNoPipeFound() throws Exception {
```

Comment:
https://github.com/snowflakedb/snowflake-ingest-java/runs/4231421925?check_suite_focus=true

The APIs wont work now because they are not enabled for this account. 

Lets ignore(@Ignore annotation from Junit) this tests now.. But lets verify if they are successful against a dev/reg environment.